### PR TITLE
add reviewers to MergeRequestEvent.java

### PR DESCRIPTION
**Pull request by mifitous**
This PR will add the missing reviewers fields to the MergeRequestEvent

- ChangeContainer<List<Reviewer>> reviewers at top level of MergeRequestEvent.java
https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#merge-request-events

Fix for issue https://github.com/gitlab4j/gitlab4j-api/issues/820

### CODE REVIEW

**Comment by mifitous** [comment]
@jetersen / @jmini :
Please kindly can you merge this one as well?
Than you a lot

**Review by jetersen — APPROVED** [review]

File: src/main/java/org/gitlab4j/api/webhook/EventChanges.java [hunk 1]
```diff
@@ -22,6 +23,7 @@ public abstract class EventChanges {
     private ChangeContainer<Long> milestoneId;
     private ChangeContainer<List<EventLabel>> labels;
     private ChangeContainer<List<Assignee>> assignees;
+    private ChangeContainer<List<Reviewer>> reviewers;
```
Comment (jmini) [inline src/main/java/org/gitlab4j/api/webhook/EventChanges.java:26]: I am not sure this is correct here.

This class is shared between `IssueChanges` and `MergeRequestChanges` and the `reviewers` attribute is something that only makes sense in the context of a MR.

So I think it should be added only to `MergeRequestChanges`.

**Review by jmini — CHANGES_REQUESTED** [review]
Please update the example file:

https://github.com/gitlab4j/gitlab4j-api/blob/69ba436c71f9d76109a957c70b8fae0577cfacb8/src/test/resources/org/gitlab4j/api/merge-request-event.json#L132

By adding a new `reviewers` attribute:

```
  "changes": {
    ....
    "reviewers": {
      "previous": [
        {
          "id": 2,
          "name": "John Doe",
          "username": "jdoe",
          "avatar_url": "https://www.gravatar.com/avatar/0?s=80&d=identicon",
          "email": "jdoe@example.com"
        }
      ],
      "current": [
        {
          "id": 3,
          "name": "Paul Smith",
          "username": "psmith",
          "avatar_url": "https://www.gravatar.com/avatar/0?s=80&d=identicon",
          "email": "psmith@example.com"
        }
      ]
    }
  },
```

**Comment by mifitous** [comment]
Hi @jmini : changes implemented
Thanks for your feedback

**Review by jetersen — APPROVED** [review]

**Review by jmini — APPROVED** [review]

**Comment by mifitous** [comment]
Thanks @jmini 
Can you please trigger a release of a new version? 5.2.1/5.3.0?
Thank you

**Pull request merged**